ledger/src/blockstore_processor.rs

```diff
@@ -3592,13 +3592,14 @@ pub mod tests {
         };
         let recyclers = VerifyRecyclers::default();
         process_bank_0(&bank0, &blockstore, &opts, &recyclers, None, None);
-        let bank1 = bank_forks.insert(Bank::new_from_parent(&bank0, &Pubkey::default(), 1));
+        let bank0_last_blockhash = bank0.last_blockhash();
```

Comment:
I generally agree that naming intermediate computation results can make the code easier to read.  Or, sometimes, more performant.
But this rename literally changes a `.` to an `_`.
Is it really worth it?